gym_cap/envs/cap_env.py

```diff
@@ -385,7 +388,35 @@ def seed(self, seed=None):
         """
         self.np_random, seed = seeding.np_random(seed)
         return [seed]
-
+        
+    def global_memory(self):
```

Comment:
Change method name to 'update global memory'

Reply:
changed accordingly